app/src/main/java/com/infomaniak/drive/utils/Extensions.kt

```diff
@@ -103,6 +105,10 @@ typealias IOFile = java.io.File
 typealias IsComplete = Boolean
 typealias Position = Int
 
+fun getAvailableStorageInBytes(): Long = with(StatFs(Environment.getDataDirectory().path)) {
+    return availableBlocksLong * blockSizeLong
```

Comment:
```suggestion
    return@with availableBlocksLong * blockSizeLong
```

Reply:
Any reason to specify the `@with` here ? We only have 1 line in this function.

Reply:
If this code ever moves in the future, the `@with` will help to not make an error and wrongly return to an incorrect scope.

I know it's pretty trivial here, but it's a matter of habit.